### PR TITLE
Discord link added.

### PR DESCRIPTION
**Pull request by KukretiShubham**

### CODE REVIEW

File: .github/workflows/invitation.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
           organization: web3community 
           label: github-invitation
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-          comment: '![welcome](https://user-images.githubusercontent.com/73097560/121835696-761b5c80-cd04-11eb-9d9c-6a4f6fb07724.gif)<br><b>Invitation sent for the GitHub Organization. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public,You can see how to do it [here](https://github.com/web3community/support/blob/main/README.md)'
+          comment: '![welcome](https://user-images.githubusercontent.com/73097560/121835696-761b5c80-cd04-11eb-9d9c-6a4f6fb07724.gif)<br><b>Invitation sent for the GitHub Organization. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public,You can see how to do it [here](https://github.com/web3community/support/blob/main/README.md)<br></br>You can also join our [Discord](https://discord.gg/wh9FwjR5j7) server'
```
Comment (vinzvinci) [inline .github/workflows/invitation.yml:15]: **you** can see how to do it here

> can you lowercase that Y? just like I did ☝️

**Comment by KukretiShubham** [comment]
@vinzvinci check now

**Review by vinzvinci — CHANGES_REQUESTED** [review]
No not that. Copy this

`welcome<br><b>Invitation sent for the GitHub Organization. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public, you can see how to do it [here](https://github.com/web3community/support/blob/main/README.md)<br></br>You can also join our [Discord](https://discord.gg/wh9FwjR5j7) server'`

**Review by shuklaritvik06 — APPROVED** [review]
LGTM

**Review by vinzvinci — APPROVED** [review]

**Pull request merged**